apps/app/src/main/scala/org/lfdecentralizedtrust/splice/config/ConfigTransforms.scala

```diff
@@ -336,6 +339,12 @@ object ConfigTransforms {
   def updateInitialTickDuration(tick: NonNegativeFiniteDuration): ConfigTransform = {
     ConfigTransforms.updateAllSvAppFoundDsoConfigs_(
       _.copy(initialTickDuration = tick)
+    ) compose ConfigTransforms.updateAllAutomationConfigs(config =>
+      if (config.pollingInterval.toInternal > tick.toInternal)
```

Comment:
we do need this or for tests that use 500ms tick we might not run triggers at all during a round

Reply:
We do however check the closing time in the round based triggers, so it messes with those triggers. 
Do you see downsides from ensuring we run during the actual interval for every round?